### PR TITLE
Update assertions in `test_pipeline_generation.py`

### PR DESCRIPTION
**Pull request by virenvarma007**
Cleared issue datajoint/element-calcium-imaging#139 by making the code update. And cleared 2022-allen-institute-workshop.ipynb files keyboard interrupt error that made it to the final branch in the notebook to clean it up a bit.

### CODE REVIEW

**Comment by kabilar** [comment]
@virenvarma007 Thanks for the discussion last week regarding this pull request.  I will provide specific suggestions in the next 1-2 weeks.

**Comment by virenvarma007** [comment]
Sure Sure Sir, No Problem

File: notebooks/2022-allen-institute-workshop.ipynb
Comment (kabilar) [inline notebooks/2022-allen-institute-workshop.ipynb:1]: We recently removed this notebook from the repository to simplify the tutorials.  Please pull from the upstream repository to fast forward your fork.

File: tests/test_pipeline_generation.py [hunk 1]
```diff
@@ -8,13 +8,8 @@ def test_generate_pipeline(pipeline):
...
     Equipment = pipeline["Equipment"]
 
-    subject_tbl, *_ = session.Session.parents(as_objects=True)
-
-    # test elements connection from lab, subject to Session
-    assert subject_tbl.full_table_name == subject.Subject.full_table_name
-
-    # test elements connection from Session to scan, imaging
-    session_tbl, equipment_tbl, _ = scan.Scan.parents(as_objects=True)
+    assert schema.Table.full_table_name in schema.OtherTable.parents()
```
Comment (kabilar) [inline tests/test_pipeline_generation.py:11]: Thanks @virenvarma007.  The code snippet provided in #74 was a template and will need to be adapted to the specific tables being tested.

**Review by kabilar — CHANGES_REQUESTED** [review]
@virenvarma007  Thanks for your contributions!  Once you make updates based on my suggestions above, please run the pytests to ensure that all pass.  Instructions for running the pytests can be found [here](https://datajoint.com/docs/elements/developer-guide/#pytests).  Please let me know if you have any questions.

File: notebooks/2022-allen-institute-workshop.ipynb
Comment (virenvarma007) [reply notebooks/2022-allen-institute-workshop.ipynb:1]: Okay ok Sir, I'll pull from the upstream repo to fast forward the fork.

**Comment by virenvarma007** [comment]
Sure sure Sir, I'll run the tests and let you know the results soon.

**Comment by virenvarma007** [comment]
Sure sure Sir, I'll run the tests and let you know the results soon.

**Pull request closed without merging**